__tests__/app.test.js

```diff
@@ -58,6 +58,7 @@ describe("GET /api/articles/:article_id", () => {
         expect(typeof body.author).toBe("string");
         expect(typeof body.title).toBe("string");
         expect(typeof body.article_id).toBe("number");
+        expect(typeof body.comment_count).toBe("string");
```

Comment:
We shouldn't really amend old tests when we get a new feature request. This old test should still pass if it's robust and we add a new test just checking for the comment_count property